universal-login-react/src/ui/Notifications/EmojiInput.tsx

```diff
@@ -0,0 +1,50 @@
+import {EmojiPlaceholders} from './EmojiPlaceholders';
+import React from 'react';
+import {EmojiKeyboard} from './EmojiKeyboard';
+import {EmojiPanelWithFakes} from './EmojiPanelWithFakes';
+import {SECURITY_CODE_LENGTH} from '@universal-login/commons';
+
+export interface EmojiInputProps {
+  value: number[];
+  onChange: (value: number[]) => void;
+  hint?: string;
```

Comment:
Naming seems wrong to me. What is "hint"? This is a public key, and if it is provided then we display a hint (which is `EmojiPanelWithFakes`), right?
How about renaming `hint` to `publicKey` / `solePublicKey` / `soleAddress`?

Reply:
Renamed